include/git2/types.h

```diff
@@ -59,7 +59,7 @@ typedef __haiku_std_int64 git_time_t;
  * app, even though /we/ define _FILE_OFFSET_BITS=64.
  */
 typedef int64_t git_off_t;
-typedef int64_t git_time_t;
+typedef int64_t git_time_t; /**< time in seconds from epoch */
```

Comment:
Didn't we have some problems correctly rendering those inline comments? Not quite sure whether I remember correctly

Reply:
I think I had this fixed on my tweaked docurium, but it's currently complaing (loudly) about `/var/folders/f4/b3wl780s12gfc25kpvl1rhk40000gn/T/HEAD-20190128-84175-yftmsd/git2/common.h:10:10: fatal error: 'time.h' file not found`, which I'll have to investigate.

Reply:
So, I can confirm that vanilla & tweaked docurium completely ignores "primitive" typedefs. I can tweak it so it doesn't (modulo more fixes to grab the correct comment), but it means I'll pickup all "primitive" typedefs (like our `size_t` `DOCURIUM` override, or the typedef goop inside our Win32 `inttypes.h`)